app/models/client.rb

```diff
@@ -456,6 +456,10 @@ def is_available_for_matching? # rubocop:disable Naming/PredicateName
     available
   end
 
+  def unavailable_fors_include_active_match
```

Comment:
maybe:
```suggestion
  def unavailable_fors_include_active_match?
```
Just to indicate it returns a boolean?

Reply:
This has been included in the lates commit